### PR TITLE
ExpandableCalendar - day selection

### PR DESCRIPTION
**Pull request by Inbal-Tish**
ExpandableCalendar - selection made from Day.state prop instead of `markedDates`

### CODE REVIEW

File: src/expandableCalendar/index.tsx [hunk 1]
```diff
@@ -172,6 +166,11 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
   const numberOfWeeks = useRef(getNumberOfWeeksInMonth(date));
 
   /** Position */
+  const [position, setPosition] = useState(initialPosition);
+  
+  const isOpen = useMemo(() => {
+    return position === Positions.OPEN;
+  }, [position]);
```
Comment (ethanshar) [inline src/expandableCalendar/index.tsx:173]: No need to memoize boolean values

File: src/expandableCalendar/WeekCalendar/index.tsx [hunk 2]
```diff
@@ -150,7 +150,6 @@ class WeekCalendar extends Component<WeekCalendarProps, State> {
         current={item}
         firstDay={firstDay}
         style={this.getWeekStyle(this.containerWidth, style)}
-        markedDates={markedDates}
```
Comment (ethanshar) [inline src/expandableCalendar/WeekCalendar/index.tsx:153]: I see u remove markDates completely, don't we still need them for general markings?

Comment (Inbal-Tish) [reply src/expandableCalendar/WeekCalendar/index.tsx:153]: They will be passed in `...others`

File: src/expandableCalendar/index.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/index.tsx:173]: Fixed

**Review by ethanshar — APPROVED** [review]

**Pull request merged**